spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -736,6 +736,50 @@
           end
         end
       end
+
+      context "SLA columns" do
```

Comment:
this controller spec mostly checks the DOM, which normally i'd do in a feature spec but the one we have (`clients_searching_sorting_and_filtering_spec`) is kind of a mess imo so i found it easier to start somewhere else where i could keep it focused on the two table columns i was trying to test